query/src/main/scala/filodb/query/ProtoConverters.scala

```diff
@@ -254,7 +255,8 @@ object ProtoConverters {
         processFailure = if (gpp.hasProcessFailure) gpp.getProcessFailure else pp.processFailure,
         processMultiPartition = if (gpp.hasProcessMultiPartition) gpp.getProcessMultiPartition
         else pp.processMultiPartition,
-        allowPartialResults = if (gpp.hasAllowPartialResults) gpp.getAllowPartialResults else pp.allowPartialResults
+        allowPartialResults = if (gpp.hasAllowPartialResults) gpp.getAllowPartialResults else pp.allowPartialResults,
+        useProtoExecPlans = if (gpp.hasUseProtoExecPlans) gpp.getUseProtoExecPlans else false
```

Comment:
Should this be `pp.useProtoExecPlans` (same as `allowPartialResults` above)?

Reply:
yes, it would be better like that, the logic of default false would be in one place